index.bs

```diff
@@ -815,6 +815,12 @@ An attribution source is a [=struct=] with the following items:
 :: A [=boolean=].
 : <dfn>fenced</dfn>
 :: A [=boolean=].
+: <dfn>attribution scope limit</dfn>
+:: Null or a positive 32-bit integer, number of distinct [=attribution source/attribution scopes=] allowed per [=attribution source/attribution destinations|attribution destination=] for the source's [=attribution source/reporting origin=].
+: <dfn>attribution scopes</dfn>
+:: A [=set=] of [=strings=].
+: <dfn>max event states</dfn>
+:: Null or a positive 64-bit integer, maximum number of [=trigger states=] for [=source type/event=] per [=attribution source/attribution destinations|attribution destination=] for the source's [=attribution source/reporting origin=].
```

Comment:
Why is this 64-bit if it's not specified in other algorithms? I think we should be consistent. cc @apasel422

Reply:
Done.